lib/random.php

```diff
@@ -45,6 +45,7 @@
  * PHP 7.0.0 and newer have these functions natively.
  */
 if (PHP_VERSION_ID >= 70000) {
+    class_alias('PHPUnit\\Framework\\TestCase', 'PHPUnit_Framework_TestCase');
```

Comment:
To be compatible with `php-5.2` syntax error, using the `class_alias` to let `PHPUnit_Framework_TestCase` can support on `php-7.x` version tests.